### DISCUSSION TITLE
file.edit() trigger editing in VSCode instead of vim

### DISCUSSION THREAD

**Discussion by shrektan** [REditorSupport/vscode-R#959] [Ideas]
Hi,

I used to run `file.edit()` in RStudio IDE and the file will be opened in the IDE, conviniently.

R' native GUI does the same.

However, in VSCode, when we open `radian` and run `file.edit()`, it opens a vim editor instead.

Is it possible to have the same feature like the RStudio IDE / R GUI?

Thanks.

## R GUI

<img width="753" alt="image" src="https://user-images.githubusercontent.com/8368933/152084304-9da22303-0036-4035-a02f-2efb28930518.png">

**Comment by renkun-ken** [comment]
In vscode terminal, it seems the terminal already has `EDITOR=code`. When I run `file.edit("somefile.R")`, the file is open in vscode editor.

**Reply by shrektan** [reply]
Thanks for the hint. I managed to resolve this by

1. setting the environment vairable explicitly in ~/.Renviron.
2. [installing 'code' command](https://stackoverflow.com/a/48538386) on macOS first.

Probably we should document this in wiki?
